orca/src/llm/openai.rs

```diff
@@ -39,6 +42,17 @@ pub struct Response {
     choices: Vec<Choice>,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct JsonModeResponse {
+    id: String,
```

Comment:
Whenever you do a /chat/completion reques the response model does include an id e.g: 
```json
{
  "id": "chatcmpl-8Rntkwjd0wyewaefR9tsOFdJ3YrmW",
  "object": "chat.completion",
  "created": 1701637356,
  "model": "gpt-3.5-turbo-1106",
  "choices": [
    {
      "index": 0,
      "message": {
        "role": "assistant",
        "content": "{\n  \"winner\": \"Los Angeles Dodgers\"\n}"
      },
      "finish_reason": "stop"
    }
  ],
  "usage": {
    "prompt_tokens": 31,
    "completion_tokens": 11,
    "total_tokens": 42
  },
  "system_fingerprint": "fp_eeff13170a"
}

```

So that's why I included it

Reply:
I'm still getting errors from the text completion endpoint though :/